components/DonatedItemsList.tsx

```diff
@@ -34,14 +38,22 @@ const DonatedItemsList: React.FC<DisplayDonatedItemsProps> = ({ userId }) => {
   }, []);
 
   const handleDeleteSuccess = (deleteItemId: number) => {
+    const deletedItem = storeItems.find((item) => item.id === deleteItemId);
+    setDeletedItemName(deletedItem?.item_name || 'Unknown name');
     setStoreItems(storeItems.filter((item) => item.id !== deleteItemId));
+    setDisplayDeletedItemMessage(true);
   };
 
   return (
     <div className='m-auto mt-10 w-5/6'>
       <h2 className='m-5 text-lg font-medium md:pl-20 lg:pl-40'>
         My donated items:
       </h2>
+      {displayDeletedItemMessage && (
+        <h2 className='mb-2 text-center text-xl font-bold'>
+          You have successfully deleted {deletedItemName} item
```

Comment:
purely a personal preference but I think the message could be more "human-readable"

```tsx
<h2>
   You have successfully deleted "{deletedItemName}"
</h2>
```